src/doc/rustdoc/src/linking-to-items-by-name.md

```diff
@@ -89,6 +114,32 @@ struct Foo {}
 fn Foo() {}
 ```
 
-Note: Because of how `macro_rules!` macros are scoped in Rust, the intra-doc links of a `macro_rules!` macro will be resolved [relative to the crate root][#72243], as opposed to the module it is defined in.
+## Warnings, re-exports, and scoping
+
+Links are resolved in the current module scope, even when re-exported. If a link from another
+crate fails to resolve, no warning is given.
+
+When re-exporting an item, rustdoc allows additional documentation to it. That documentation will
```

Comment:
I think you're missing a word here.

```suggestion
When re-exporting an item, rustdoc allows adding additional documentation to it. That documentation will
```

Reply:
We should probably define what re-exports are or link to the `doc(inline)` page somewhere.